testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/reactive/KeystoreUtil.java

```diff
@@ -25,10 +26,16 @@ public class KeystoreUtil {
 
     private static final String KEY_STORE_DIRECTORY = "target/reactive-messaging-security";
     public static final Path KEY_STORE_DIRECTORY_PATH = Paths.get(KEY_STORE_DIRECTORY);
+
     public static final String SERVER_KEYSTORE = KEY_STORE_DIRECTORY + "/server.keystore.p12";
     public static final Path SERVER_KEYSTORE_PATH = Paths.get(SERVER_KEYSTORE);
+
+    public static final String SERVER_KEYSTORE_CREDENTIALS = KEY_STORE_DIRECTORY + "/server.keystore.pwd.txt";
+
+    public static final Path SERVER_KEYSTORE_CREDENTIALS_PATH = Paths.get(SERVER_KEYSTORE_CREDENTIALS);
+
     private static final String SERVER_CER = KEY_STORE_DIRECTORY + "/server.cer";
-    private static final String SERVER_TRUSTSTORE = KEY_STORE_DIRECTORY + "/server.truststore.p12";
+    private static final String SERVER_TRUSTSTORE = KEY_STORE_DIRECTORY + "/git ";
```

Comment:
Something is off here. The change seems inadvertent - and also, there is a trailing space.

Reply:
Thanks, I must have been trying to run terminal commands while in intellij :-)